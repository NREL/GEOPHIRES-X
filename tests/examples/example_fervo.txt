GEOPHIRES v3.0 Input File
Electrcity using a Thermal Drawdown Reservoir Model and BICYCLE Economic Model

example_fervo description: This example problem considers an L-shaped EGS reservoir at 3.1km depth.
The heat is used in electricity is made with double flash.

***Subsurface technical parameters***
*************************************
Has Nonvertical Section, True
Multilaterals Cased, True
Well Geometry Configuration, 4,       ----L
Number of Multilateral Sections, 2,  ---Two parallel horizontal sections
Total Nonvertical Length, 3005.1
Reservoir Model,3,						---m/A Single Fracture Thermal Drawdown
Well Drilling Cost Correlation,3,				--- [-] Use built-in well drilling cost correlation #3 = vertical large diameter
Horizontal Well Drilling Cost Correlation,4,				--- [-] Use built-in well drilling cost correlation #4 = deviated large diameter
Drawdown Parameter,.00002,					---[kg/s/m2]
Reservoir Depth,3.1,						---[km]
Number of Segments,1,						---[-]
Gradient 1,70,							---[deg.C/km]
Maximum Temperature,400,					---[deg.C]
Number of Production Wells,1,					---[-]
Number of Injection Wells,1,					---[-]
Production Well Diameter,8.5,					---[inch]
Injection Well Diameter,8.5,					---[inch]
Ramey Production Wellbore Model,1,				---Should be 0 (disabled) or 1 (enabled)
Production Wellbore Temperature Drop,5,				---[deg.C]
Injection Wellbore Temperature Gain,3,				---[deg.C]
Production Flow Rate per Well,70,				---[kg/s]
Reservoir Volume Option,1,					---Should be 1 2 3 or 4. See manual for details.
Fracture Shape,1,						---Should be 1 2 3 or 4
Fracture Area,200000,						---[m2]
Number of Fractures,12,						---[-]
Fracture Separation,80,						---[m]
Injectivity Index,5,						---[kg/s/bar]
Injection Temperature,70,					---[deg.C]
Maximum Drawdown,1,						---[-] no redrilling considered
Reservoir Heat Capacity,1000,					---[J/kg/K]
Reservoir Density,3000,						---[kg/m3]
Reservoir Thermal Conductivity,3,				---[W/m/K]
Water Loss Fraction,0.02,						---[-]

***Surface Technical Parameters***
**********************************

End-Use Option,1,						---electricity as the main product
Power Plant Type,4,						---Double-Flash
Circulation Pump Efficiency,.80,				---[-]
Utilization Factor,.9,						---[-]
End-Use Efficiency Factor,.9,					---[-]
Surface Temperature,15,						---[deg.C]
Ambient Temperature,15,						---[deg.C]

***Financial Parameters***
**************************

Plant Lifetime,35,						---[years]
Economic Model,3,						---BICYCLE Levelized Cost Model
Fraction of Investment in Bonds,.5,				---[-] Required for BICYCLE model
Inflated Bond Interest Rate,.05,				---[-] Required for BICYCLE model
Inflated Equity Interest Rate,.08,				---[-] Required for BICYCLE model
Inflation Rate,.02,						---[-] Required for BICYCLE model
Combined Income Tax Rate,.3,					---[-] Required for BICYCLE model
Gross Revenue Tax Rate,0,					---[-] Required for BICYCLE model
Investment Tax Credit Rate,0,					---[-] Required for BICYCLE model
Property Tax Rate,0,						---[-] Required for BICYCLE model
Inflation Rate During Construction,0.05,				---[-]
Starting Electricity Sale Price,0.15
Ending Electricity Sale Price,1.00
Electricity Escalation Start Year,1
Electricity Escalation Rate Per Year, 0.004053223


***Simulation Parameters***
***************************

Print Output to Console,1,					---Should be 1 (to print) or 0 (to not print)
Time steps per year,10,						---[-]
