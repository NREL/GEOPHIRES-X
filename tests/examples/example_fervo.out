                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.4.25
 GEOPHIRES Build Date: 2024-03-05
 Simulation Date: 2024-05-07
 Simulation Time:  17:57
 Calculation Time:      0.340 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Electricity
      Average Net Electricity Production:                     6.64 MW
      Electricity breakeven price:                           11.66 cents/kWh
      Number of production wells:                             1
      Number of injection wells:                              1
      Flowrate per production well:                          70.0 kg/sec
      Well depth (or total length, if not vertical):          3.1 kilometer
      Geothermal gradient:                                    0.0700 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model  = BICYCLE
      Accrued financing during construction:                  5.00
      Project lifetime:                                      35 yr
      Capacity factor:                                       90.0 %
      Project NPV:                                          53.41 MUSD
      Project IRR:                                          11.68 %
      Project VIR=PI=PIR:                                    1.80
      Project MOIC:                                          2.08
      Project Payback Period:                               10.14 yr

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             1
      Number of Injection Wells:                              1
      Well depth (or total length, if not vertical):          3.1 kilometer
      Water loss rate:                                        2.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 73.0 degC
      Production Wellbore heat transmission calculated with Ramey's model
      Average production well temperature drop:               3.8 degC
      Flowrate per production well:                          70.0 kg/sec
      Injection well casing ID:                               8.500 in
      Production well casing ID:                              8.500 in
      Number of times redrilling:                             0
      Power plant type:                                       Double-Flash


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        400.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0700 degC/m


                           ***RESERVOIR PARAMETERS***

      Reservoir Model = Single Fracture m/A Thermal Drawdown Model
      m/A Drawdown Parameter:                                 0.00002 1/year
      Bottom-hole temperature:                              232.00 degC
      Reservoir volume calculated with fracture separation and number of fractures as input
      Number of fractures:                                   12.00
      Fracture separation:                                   80.00 meter
      Reservoir volume:                               176000000 m**3
      Reservoir hydrostatic pressure:                         29742.69 kPa
      Plant outlet pressure:                                100.00 kPa
      Injectivity Index:                                      5.00 kg/sec/bar
      Reservoir density:                                   3000.00 kg/m**3
      Reservoir thermal conductivity:                         3.00 W/m/K
      Reservoir heat capacity:                             1000.00 J/kg/K


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       227.7 degC
      Average Production Temperature:                       222.0 degC
      Minimum Production Temperature:                       210.2 degC
      Initial Production Temperature:                       224.9 degC
      Average Reservoir Heat Extraction:                     42.61 MW
      Production Wellbore Heat Transmission Model = Ramey Model
      Average Production Well Temperature Drop:               3.8 degC
      Average Injection Well Pump Pressure Drop:            735.4 kPa


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      30.04 MUSD
             Drilling and completion costs per vertical production well:         7.29 MUSD
             Drilling and completion costs per vertical injection well:          7.29 MUSD
             Drilling and completion costs per non-vertical sections:           14.02 MUSD
         Stimulation costs:                                   1.51 MUSD
         Surface power plant costs:                          27.28 MUSD
         Field gathering system costs:                        1.11 MUSD
         Total surface equipment costs:                      28.38 MUSD
         Exploration costs:                                   6.93 MUSD
      Total capital costs:                                   66.86 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.55 MUSD/yr
         Power plant maintenance costs:                       1.11 MUSD/yr
         Water costs:                                         0.04 MUSD/yr
      Total operating and maintenance costs:                  1.69 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Initial geofluid availability:                          0.23 MW/(kg/s)
      Maximum Total Electricity Generation:                   7.11 MW
      Average Total Electricity Generation:                   6.70 MW
      Minimum Total Electricity Generation:                   5.89 MW
      Initial Total Electricity Generation:                   6.90 MW
      Maximum Net Electricity Generation:                     7.04 MW
      Average Net Electricity Generation:                     6.64 MW
      Minimum Net Electricity Generation:                     5.82 MW
      Initial Net Electricity Generation:                     6.84 MW
      Average Annual Total Electricity Generation:           52.68 GWh
      Average Annual Net Electricity Generation:             52.17 GWh
      Initial pumping power/net installed power:              0.95 %
      Average Pumping Power:                                  0.06 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR       THERMAL               GEOFLUID               PUMP               NET               FIRST LAW
             DRAWDOWN             TEMPERATURE             POWER             POWER              EFFICIENCY
                                     (degC)               (MW)              (MW)                  (%)
   1           1.0000                224.91               0.0648            6.8366               15.7372
   2           1.0085                226.82               0.0648            6.9753               15.8572
   3           1.0102                227.20               0.0648            7.0037               15.8817
   4           1.0111                227.41               0.0648            7.0185               15.8944
   5           1.0117                227.54               0.0648            7.0282               15.9028
   6           1.0121                227.63               0.0648            7.0350               15.9086
   7           1.0124                227.69               0.0648            7.0392               15.9123
   8           1.0124                227.70               0.0648            7.0403               15.9132
   9           1.0123                227.66               0.0648            7.0373               15.9106
  10           1.0118                227.56               0.0648            7.0295               15.9039
  11           1.0110                227.38               0.0648            7.0165               15.8927
  12           1.0099                227.13               0.0648            6.9982               15.8769
  13           1.0084                226.81               0.0648            6.9745               15.8565
  14           1.0067                226.41               0.0648            6.9458               15.8318
  15           1.0047                225.95               0.0648            6.9125               15.8030
  16           1.0024                225.44               0.0648            6.8750               15.7705
  17           0.9998                224.87               0.0648            6.8338               15.7348
  18           0.9971                224.25               0.0648            6.7894               15.6963
  19           0.9942                223.60               0.0648            6.7423               15.6553
  20           0.9911                222.90               0.0648            6.6929               15.6122
  21           0.9879                222.18               0.0648            6.6416               15.5674
  22           0.9846                221.44               0.0648            6.5887               15.5211
  23           0.9812                220.67               0.0648            6.5347               15.4737
  24           0.9777                219.89               0.0648            6.4797               15.4254
  25           0.9741                219.09               0.0648            6.4241               15.3764
  26           0.9706                218.29               0.0648            6.3681               15.3269
  27           0.9669                217.47               0.0648            6.3118               15.2770
  28           0.9633                216.65               0.0648            6.2555               15.2270
  29           0.9596                215.83               0.0648            6.1992               15.1769
  30           0.9560                215.01               0.0648            6.1432               15.1269
  31           0.9523                214.18               0.0648            6.0874               15.0771
  32           0.9487                213.36               0.0648            6.0321               15.0275
  33           0.9450                212.54               0.0648            5.9773               14.9782
  34           0.9414                211.73               0.0648            5.9230               14.9293
  35           0.9378                210.92               0.0648            5.8692               14.8808


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR             ELECTRICITY                   HEAT                RESERVOIR            PERCENTAGE OF
                    PROVIDED                   EXTRACTED            HEAT CONTENT        TOTAL HEAT MINED
                   (GWh/year)                  (GWh/year)            (10^15 J)                 (%)
   1                  54.6                       345.3                 82.71                   1.48
   2                  55.1                       347.3                 81.46                   2.97
   3                  55.3                       347.9                 80.21                   4.46
   4                  55.4                       348.3                 78.95                   5.96
   5                  55.4                       348.5                 77.70                   7.45
   6                  55.5                       348.7                 76.44                   8.95
   7                  55.5                       348.8                 75.19                  10.44
   8                  55.5                       348.8                 73.93                  11.94
   9                  55.5                       348.6                 72.68                  13.43
  10                  55.4                       348.3                 71.42                  14.93
  11                  55.2                       347.8                 70.17                  16.42
  12                  55.1                       347.2                 68.92                  17.91
  13                  54.9                       346.3                 67.67                  19.39
  14                  54.6                       345.4                 66.43                  20.87
  15                  54.4                       344.3                 65.19                  22.35
  16                  54.0                       343.1                 63.96                  23.82
  17                  53.7                       341.7                 62.73                  25.28
  18                  53.3                       340.3                 61.50                  26.74
  19                  53.0                       338.8                 60.28                  28.20
  20                  52.6                       337.2                 59.07                  29.64
  21                  52.2                       335.5                 57.86                  31.08
  22                  51.7                       333.8                 56.66                  32.51
  23                  51.3                       332.1                 55.46                  33.94
  24                  50.9                       330.3                 54.27                  35.35
  25                  50.4                       328.5                 53.09                  36.76
  26                  50.0                       326.7                 51.91                  38.16
  27                  49.5                       324.8                 50.75                  39.55
  28                  49.1                       323.0                 49.58                  40.94
  29                  48.7                       321.1                 48.43                  42.32
  30                  48.2                       319.2                 47.28                  43.69
  31                  47.8                       317.4                 46.13                  45.05
  32                  47.3                       315.5                 45.00                  46.40
  33                  46.9                       313.7                 43.87                  47.74
  34                  46.5                       311.9                 42.75                  49.08
  35                  41.5                       279.1                 41.74                  50.28


                             ********************************
                             *  REVENUE & CASHFLOW PROFILE  *
                             ********************************
Year            Electricity             |            Heat                  |           Cooling                 |         Carbon                    |          Project
Since     Price   Ann. Rev.  Cumm. Rev. |   Price   Ann. Rev.   Cumm. Rev. |  Price   Ann. Rev.   Cumm. Rev.   |   Price   Ann. Rev.   Cumm. Rev.  | OPEX    Net Rev.      Net Cashflow
Start    (cents/kWh)(MUSD/yr) (MUSD)    |(cents/kWh) (MUSD/yr)    (MUSD)   |(cents/kWh) (MUSD/yr)    (MUSD)    |(USD/tonne) (MUSD/yr)    (MUSD)    |(MUSD/yr) (MUSD/yr)    (MUSD)
________________________________________________________________________________________________________________________________________________________________________________________
  1      0.00          -66.86   0.00     |    0.00     0.00         0.00    |    0.00     0.00         0.00     |    0.00     0.00         0.00     |  0.00     -66.86     -66.86
  2     15.00           6.50   8.19     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      6.50     -60.36
  3     15.00           6.57  16.46     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      6.57     -53.79
  4     15.41           6.82  24.98     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      6.82     -46.96
  5     15.81           7.06  33.73     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      7.06     -39.90
  6     16.22           7.30  42.72     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      7.30     -32.61
  7     16.62           7.53  51.94     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      7.53     -25.08
  8     17.03           7.76  61.39     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      7.76     -17.32
  9     17.43           7.98  71.07     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      7.98     -9.34
 10     17.84           8.20  80.96     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      8.20     -1.14
 11     18.24           8.41  91.06     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      8.41      7.27
 12     18.65           8.61  101.36     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      8.61     15.88
 13     19.05           8.80  111.86     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      8.80     24.68
 14     19.46           8.98  122.54     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      8.98     33.66
 15     19.86           9.16  133.39     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      9.16     42.82
 16     20.27           9.32  144.41     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      9.32     52.15
 17     20.67           9.48  155.58     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      9.48     61.62
 18     21.08           9.63  166.90     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      9.63     71.25
 19     21.49           9.77  178.36     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      9.77     81.02
 20     21.89           9.90  189.95     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69      9.90     90.92
 21     22.30          10.03  201.67     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     10.03     100.95
 22     22.70          10.15  213.51     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     10.15     111.09
 23     23.11          10.26  225.47     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     10.26     121.35
 24     23.51          10.37  237.53     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     10.37     131.72
 25     23.92          10.47  249.70     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     10.47     142.20
 26     24.32          10.57  261.96     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     10.57     152.77
 27     24.73          10.67  274.32     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     10.67     163.43
 28     25.13          10.76  286.77     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     10.76     174.19
 29     25.54          10.85  299.31     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     10.85     185.04
 30     25.94          10.93  311.93     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     10.93     195.97
 31     26.35          11.01  324.64     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     11.01     206.98
 32     26.75          11.09  337.42     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     11.09     218.06
 33     27.16          11.16  350.28     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     11.16     229.23
 34     27.56          11.24  363.21     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     11.24     240.47
 35     27.97          11.31  376.21     |    2.50     0.00         0.00    |    2.50     0.00         0.00     |    0.00     0.00         0.00     |  1.69     11.31     251.77
